src/context/providers/PointsGridFormDataProvider.js

```diff
@@ -30,6 +30,10 @@ export class PointsGridFormDataProvider extends mix(JSONSchemaFormDataProvider).
         return 0;
     }
 
+    static dimensionsHaveNumbers(dimensions) {
```

Comment:
This is the time when I'm not sure we need this as a separate function...

Reply:
it was mostly for the purpose of readability, but I'll remove it then

Reply:
ok after refactoring it has almost the same length as the original function call 😄